app/views/trainees/check_details/_error_summary.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
```

Comment:
@edwardhorsford - thought I'd highlight that I've had to render this error summary manually, since I can't use the form builder's `govuk_error_summary` method.

I don't imagine we'll ever use this error summary anywhere else in the code, so I haven't created a generic component.

What are your thoughts?